values-prereqs.yaml

```
@@ -18,6 +18,15 @@ bootstrap:
   initialAdminUser:
     password:
       value: "admin"
+  db:
+    cluster:
+      password:
+        valueFrom:
+          secretKeyRef:
+            name: "prereqs-postgresql"
+            key: "password"
+      username:
+        value: "user"
```

Just to keep people on their toes, I named these differently in the new `corda-dev-prereqs` chart. You should check that these overrides work there.
```suggestion
            name: "prereqs-postgres"
            key: "postgres-password"
      username:
        value: "postgres"
```